### PR TITLE
Gateway skip verify in Application CR

### PR DESCRIPTION
**Pull request by koala7659**
Fix for the issue : [Disabling SSL certificate verification per Application with Central App Gateway #11946
](https://github.com/kyma-project/kyma/issues/11946)

- [x] Extending Application CR with new `SkipVerify` flag
- [x] `Central Application Gateway. `Processing `SkipVerify` flag from Application CR to create proxy cache entry for each API. Removing old parameter from configuration.
- [x] `Compass Runtime Agent.` New configuration parameter to setup the default value of `skipVerify` flag for each created Application CR from Director
- [x] Application Registry. Ensure `skipVerify` flag is set to false for each created Application CR
- [x] Build problem to fix - imports are not correct

### CODE REVIEW

File: components/central-application-gateway/README.md [hunk 1]
```diff
@@ -43,7 +43,6 @@ The Central Application Gateway has the following parameters:
 - **externalAPIPort** is the port that exposes the API which allows checking the component status. The default port is `8081`.
 - **applicationSecretsNamespace** is the Namespace in which the Application secrets used by the Central Application Gateway exist. The default Namespace is `kyma-integration`.
 - **requestTimeout** is the timeout for requests sent through the Central Application Gateway, expressed in seconds. The default value is `1`.
-- **skipVerify** is the flag for skipping the verification of certificates for the proxy targets. The default value is `false`.
```
Comment (majakurcius) [inline components/central-application-gateway/README.md:46]: Are we sure we want to remove it?

File: components/compass-runtime-agent/README.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ The Runtime Agent uses these environment variables:
 - **APP_CLUSTER_CERTIFICATES_SECRET** specifies the Namespace and the Name of the Secret in which to store the client certificate and the key.
 - **APP_CA_CERTIFICATES_SECRET** specifies the Namespace and the Name of the Secret in which to store the CA certificate.
 - **APP_SKIP_COMPASS_TLS_VERIFY** specifies whether to communicate with the Connector Service and the Director with disabled TLS verification.
+- **APP_SKIP_APPS_TLS_VERIFY** specifies whether applications synchronized from Compass will be setup to communicate with external systems with disabled TLS verification.
```
Comment (majakurcius) [inline components/compass-runtime-agent/README.md:35]:
```suggestion
- **APP_SKIP_APPS_TLS_VERIFY** specifies whether applications synchronized from Compass will be set up to communicate with external systems with disabled TLS verification.
```
- `setup` is a noun, the verb is `set up`. 
- By `application` we DON'T mean the representation of the external system here, right?

**Comment by franpog859** [comment]
There is a problem with deploying Application Connector in this stage of the implementation. I'm on it
```
...
X Component 'application-connector' failed 
context deadline exceeded
X Deploying Kyma
Deployed Components: 19/20
Could not deploy the following components:
- application-connector
...
```

**Comment by franpog859** [comment]
Problem mentioned in [this comment](https://github.com/kyma-project/kyma/pull/12257#issuecomment-1021044349) is fixed in [this PR](https://github.com/kyma-project/kyma/pull/13170)

**Review by Maladie — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]

**Review by pPrecel — APPROVED** [review]

**Review by jeremyharisch — APPROVED** [review]

**Pull request merged**